book/doctrine.rst

```diff
@@ -738,8 +749,6 @@ Doctrine's native SQL-like language called DQL to make a query for this::
     )->setParameter('price', '19.99');
 
     $products = $query->getResult();
```

Comment:
I removed lines 741-742 because the `getOneOrNullResult()` method is introduced in more detail further down in this document on line 768.  Including it here is a little premature/redundant.